csrc/codebase/mmseg/segment.cpp

```diff
@@ -68,11 +68,20 @@ class ResizeMask : public MMSegmentation {
       return to_value(output);
     } else {
       cv::Mat _dst;
-      cv::extractChannel(dst, _dst, 0);
+      int channel = IsLittleEndian() ? 0 : dst.dims - 1;
+      cv::extractChannel(dst, _dst, channel);
       auto output_tensor = cpu::CVMat2Tensor(_dst);
       SegmentorOutput output{output_tensor, dst_height, dst_width, classes_};
       return to_value(output);
     }
+
+    bool IsLittleEndian() {
```

Comment:
Where is this comming from?

Reply:
updated

Reply:
let's convert it to a const boolean member since endianness will stay const for a single setting.